cannelloni.cpp

```diff
@@ -124,9 +128,10 @@ int main(int argc, char** argv) {
     }
   }
   if (!remoteIPSupplied) {
-    std::cout << "Error: Remote IP not supplied" << std::endl;
-    printUsage();
-    return -1;
+    std::cout << "Remote IP not supplied , will be bound to the first incoming connection" << std::endl;
```

Comment:
`linfo << "Remote IP not supplied , will bind to the first incoming connection" << std::endl;